src/main/java/reposense/model/AuthorConfiguration.java

```diff
@@ -213,8 +228,16 @@ public void setAuthorDisplayName(Author author, String displayName) {
         authorDisplayNameMap.put(author, displayName);
     }
 
-    public void addAuthorEmailsAndAliasesMapEntry(Author author, List<String> values) {
-        values.forEach(value -> authorEmailsAndAliasesMap.put(value, author));
+    /**
+     * Adds {@code aliases} of {@code author} into the map
+     * @param author
+     * @param aliases
+     */
+    public void addAuthorEmailsAndAliasesMapEntry(Author author, List<String> aliases) {
```

Comment:
why the change from values -> aliases? If I remember correctly, the map is not just `alias -> author` right?